components/Asana-Card/AsanaCard.js

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export default function AsanaCard({ name, img }) {
+  return (
+    <StyledArticle>
+      <ImageWrapper>
+        <Image
+          src={img}
+          alt={name}
+          width={1000}
+          height={1000}
+          layout="responsive"
+        />
+      </ImageWrapper>
+      <h2>{name}</h2>
+    </StyledArticle>
+  );
+}
+
+const ImageWrapper = styled.section`
+  width: 100px;
+  height: 100px;
+`;
```

Comment:
Why don't you specify this width in the Image itself? This should be a figure instead of a section.

Reply:
True ;)